main.html

```diff
@@ -1,77 +1,255 @@
+    </p>
+    <p>
+      This option sounds attractive for the consuming team because they get the requested feature without doing the work to implement or maintain it.
+      It is still a drag on the team, though, because it necessarily diverts some of their attention and work to the non-productive task of escalation.
+      Additionally, this option does not scale as there are so many times that a consumer can escalate feature requests before damaging their credibility.
+    </p>
+    <p>
+      <em>Escalate</em> is similarly disruptive (even more so) for the producing team, who are taken out of their normal workflow and prioritization methods to deal with the escalated feature request.
+    </p>
+  </li>
+</ol>
+
+<pre>
+  The "Escalate" option could be a place to include work for Danese's "cheeses" diagram.
+  "Workaround" could possibly use a diagram as well.
 </pre>
+
+<h2>Solution Overview</h2>
+
+<p>
+  This Oriole explains a fourth option available to the consuming team - <strong>InnerSource</strong>.
+  This strategy provides the wins of <em>Workaround</em> and <em>Escalate</em> without the same drawbacks to the teams and company.
+</p>
+<p>
+  The basic idea of <em>InnerSource</em> is this: If a consuming team can't get what it needs by submitting a feature request then it may submit a pull request instead.
+  The producing team can partner to review and accept the pull request and the functionality it implements.
+</p>
+<p>
+  At its best, this option works for the consuming team because they get the requested feature without taking on the long-term burden of its maintenance.
+  It works for the producing team because they are able to better scale and serve their consumers.
+  It works for the company at-large because solutions to shared problems end up in shared, centrally-maintained locations where all can use them. 
+  Additionally, engineering focus stays on producing the code that solves company problems rather than the mechanics of the feature negotiation and escalation process.
+</p>
+
+<h2>Solution Detail</h2>
+<p>
+  Let's take a closer look at how the mechanics of InnerSource can work.
+  To aid in this explanation, we'll define a few key terms.
+  First, in the context of the code contribution process, we call the team that is receiving the code the <strong>Host team</strong>.
+  Conversely, the team giving the code is the <strong>Guest team</strong>.
+  There are also a few key individuals to name.
+  First, the <strong>Product Owner</strong> determines what functionality the host team is willing to accept as contributions.
+  The <strong>Contributor</strong> is the individual submitting the code contribution.
+  The <strong>Trusted Committer</strong> provides any timely support and mentorship that the contributor needs to successfully submit the code contribution.
+</p>
+
+<p>
+  With those definitions, here is the basic outline of an InnerSource contribution.
+</p>
+<p>
+  <ol>
```

Comment:
I don't think an `<ol>` is allowed to be a child of a `<p>`, ref: https://stackoverflow.com/a/10601411

Reply:
Fixed, Nick.  Thanks for keeping the review technical ❤️ 👍 .